tests/ops/op_math/test_sprod.py

```diff
@@ -667,3 +667,22 @@ def my_circ():
 
         with pytest.raises(QuantumFunctionError, match="SProd is not an observable:"):
             my_circ()
+
+    @pytest.mark.torch
```

Comment:
Worth adding tests for tf and jax?

Reply:
Can't hurt.  This was mostly to check the tape could be unwrapped to numpy without errors.